src/pages/Trash/Trash.js

```diff
@@ -0,0 +1,25 @@
+import { Header, NavBar,TrashNote } from "../../components/index"
+import {useTrash} from "../../context"
+import "../commonPage.css"
+import "./trash.css"
+
+export const Trash = () => {
+    const {trashNotes} = useTrash()
+    return(
+        <>
+         <main className="main-page">
+           <Header/>
+           <NavBar/>
+           <div className="page-content">
+                 <div className="trash-notes">
+                     {
+                         trashNotes && trashNotes.map(trashNote => (
+                            <TrashNote key={trashNote.id} trashNote={trashNote}/>
+                         ))
```

Comment:
Same as above for archiveNotes, do check this as well before merging.

Reply:
It is only for the safer side if anything is not present then it won't render